### ISSUE TITLE
Forward logs to different systems based on their "inputSource"

### ISSUE THREAD

**Issue by SAbdulRahuman** [openshift/cluster-logging-operator#777]
I see, In OCP 4.6, Log Forwarding API is supported and logs can be forwarded to different systems based on their "inputSource".

![image](https://user-images.githubusercontent.com/7375503/98546705-1f0c5f80-22bd-11eb-812c-5a3aa6f37239.png)
[PPT link](https://speakerdeck.com/redhatopenshift/whats-new-in-openshift-container-platform-4-dot-6?slide=18)

I have a doubt here, Could anyone clarify me ?.

If logs are tagged/labeled like "KEY:VALUE" (i.e, "security":"enabled"), then how to forwards those logs (which has labed like secutiry:enabled) to seperate systems ?.

In other words, How to forwards logs to different systems (i.e, elasticsearch) based on labels in logs ?.

**Comment by jcantrill** [comment]
"InputSource" is an opinionated reference, not configurable by label https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-log-forwarding.md#proposal
